elastic/kibana
# [Alerting UI] Fixing behavior when trying to render an Index Threshold visualization with invalid data

Resolves https://github.com/elastic/kibana/issues/99228

## Summary

The visualization component was trying to render the full error object returned when the request to get visualization failed. This PR fixes the behavior to render just the error message. I also removed the toaster notification because it seemed redundant with the error callout.

## To verify

Start creating an index threshold rule. Change Group By field to `top`, select a field and enter 1001. This will return an error from the visualization API because 1000 is greater than the max allowed value. The create flyout should not go into an error state and instead show an error callout with an appropriate error message.

### Checklist
